1/test/run

```diff
@@ -11,6 +11,8 @@ shopt -s nullglob
 
 IMAGE_NAME=${IMAGE_NAME-openshift/jenkins-1-centos7-candidate}
 
+FAILURE=`echo true`
```

Comment:
It was causing funny highlighting in my emacs bash editor and i wanted to make sure i got something functional to test ... I'll work on switching